defaults/clade_display_names.yml

```diff
@@ -0,0 +1,32 @@
+19B: 19B
```

Comment:
Is there a reason there's no `19A: 19A`?

Reply:
none that I can think of ;)